portable/GCC/NiosII/port_asm.S

```diff
@@ -58,7 +57,8 @@ save_context:
 	stw		r15, 64(sp)
 	rdctl	r5, estatus 		# Save the eStatus
 	stw		r5, 68(sp)
-	stw		ea, 72(sp)			# Save the PC
+	addi	r15, ea, -4			# Instruction that caused exception

```

Comment:
I don't understand this change. Will require follow up.

Reply:
`stw ea, 72(sp)` equals to `addi r15, ea, -4`(this line) + `stw r15, 72(sp)`(next line) .  This is related to `addi ea, ea, -4` was removed. 

This expression is same with written in BSP generated code.